open_seq2seq/utils/utils.py

```diff
@@ -439,4 +449,301 @@ def cast_types(input_dict, dtype):
       cast_input_dict[key] = cur_list
       continue
     cast_input_dict[key] = input_dict[key]
-  return cast_input_dict
\ No newline at end of file
+  return cast_input_dict
+
+def get_interactive_infer_results(model, sess, model_in):
```

Comment:
Why do we need to return a list with 1 element instead of just returning that element? I mean, shouldn’t this function just return model.infer(...)?